build.psm1

```diff
@@ -19,114 +19,96 @@ $tagsUpToDate = $false
 # Since code that uses Get-PSCommitID and Get-PSLatestTag assume that tags are fetched,
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
Type extensible completion of type after using namespace.Command line: <using namespace System; [TextR>.IO.TextReader
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Parser/ExtensibleCompletion.Tests.ps1: line 112
112:                         $expected.Found | Should -BeTrue
```
Type extensible completion of type after using namespace.Command line: <using namespace System.IO; [TextR>.TextReader
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Parser/ExtensibleCompletion.Tests.ps1: line 112
112:                         $expected.Found | Should -BeTrue
```